test/e2e/suites/e2e/e2e_test.go

```diff
@@ -173,6 +176,7 @@ var _ = Describe("e2e tests", func() {
 
 			port := plist[0]
 			Expect(port.Description).To(Equal("primary"))
+			Expect(port.Tags).To(ContainElement(testTag))
```

Comment:
Hmm, this test is broken 🤔 It's dumping all ports with a description of "primary" with no other constraint. This could easily pass due to a concurrently running test creating a different port with this name.

However... Good News! We can now tag ports 😁

Please could you ensure that the test tag is unique to the test, then fetch ports by tag and ensure you get the one with description "primary"? Suggest using clusterName as a tag would work well.